application/helpers/expressions/em_manager_helper.php

```diff
@@ -8548,8 +8550,17 @@ static function ProcessCurrentResponses()
                                     }
                                 }
                                 break;
-#                            case 'N': //NUMERICAL QUESTION TYPE
-#                            case 'K': //MULTIPLE NUMERICAL QUESTION
+                            case 'N': //NUMERICAL QUESTION TYPE
+                            case 'K': //MULTIPLE NUMERICAL QUESTION
+                                if (trim($value)!="") {
+                                    $aAttributes=$LEM->getQuestionAttributesForEM($LEM->sid, $qid, $_SESSION['LEMlang']);
+                                    if (!isset($aAttributes[$qid])) {
+                                        $aAttributes[$qid]=array();
+                                    }
+                                    if($aAttributes[$qid]['thousands_separator']=='1') {
+                                        $value=str_replace($invertRadix['separator'], '', $value);
+                                    }
+                                }
```

Comment:
Ok, I found it ! :D
Line 8523 to 8527 :

    if ($radixchange && isset($LEM->knownVars[$sq]['onlynum']) && $LEM->knownVars[$sq]['onlynum']=='1')
    {
        // convert from comma back to decimal
        $value = implode('.',explode(',',$value));
    }


I just need to move this part after the "switch".